crates/biome_js_analyze/src/analyzers/nursery/no_exports_in_test.rs

```diff
@@ -0,0 +1,183 @@
+                        // module.exports = {}
+                        let is_commonjs_export = object.is_some_and(|object| match object {
+                            AnyJsExpression::JsIdentifierExpression(ident) => match member_expr {
+                                AnyJsMemberAssignment::JsComputedMemberAssignment(_) => false,
+                                AnyJsMemberAssignment::JsStaticMemberAssignment(static_member) => {
+                                    let indent_text = ident.text();
+                                    let member_text =
+                                        static_member.member().map(|member| member.text());
+                                    indent_text == "module"
+                                        && member_text
+                                            .is_ok_and(|member_text| member_text == "exports")
+                                }
+                            },
+                            _ => {
+                                // modules.exports.foo = {}, module.exports[foo] = {}
+                                let is_commonjs_export = object
+                                    .as_js_static_member_expression()
+                                    .is_some_and(|member_expr| {
+                                        let object_text =
+                                            member_expr.object().map(|object| object.text());
+                                        let member_text =
+                                            member_expr.member().map(|member| member.text());
+                                        object_text.is_ok_and(|text| text == "module")
+                                            && member_text
+                                                .is_ok_and(|member_text| member_text == "exports")
+                                    });
+                                is_commonjs_export
+                            }
+                        });
+                        is_commonjs_export
+                    });
+                is_commonjs_export
+            }
+        }
+    }
+}
+
+#[derive(Default)]
+struct AnyExportInTestVisitor {
+    has_test: bool,
+    exports: Vec<MaybeExport>,
+}
+
+impl Visitor for AnyExportInTestVisitor {
+    type Language = JsLanguage;
+
+    fn visit(
+        &mut self,
+        event: &biome_rowan::WalkEvent<biome_rowan::SyntaxNode<Self::Language>>,
+        mut ctx: biome_analyze::VisitorContext<Self::Language>,
+    ) {
+        match event {
+            WalkEvent::Enter(node) => {
+                if let Some(maybe_export) = MaybeExport::cast_ref(node) {
+                    if maybe_export.is_export() {
+                        self.exports.push(maybe_export);
+                    }
+                }
+
+                if !self.has_test {
+                    if let Some(call_expr) = JsCallExpression::cast_ref(node) {
+                        self.has_test = call_expr.is_test_call_expression().ok().unwrap_or(false);
```

Comment:
I think `.ok` can be omitted. `Result` has [`unwrap_or`](https://doc.rust-lang.org/std/result/enum.Result.html#method.unwrap_or)